src/test/java/org/example/testSteps/InventoryPageSteps.java

```diff
@@ -0,0 +1,36 @@
+package org.example.testSteps;
+
+import static com.codeborne.selenide.Condition.visible;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.example.enums.ItemsEnum;
+import org.example.pageObjects.InventoryPage;
+
+public class InventoryPageSteps {
+
+    static InventoryPage inventoryPage;
+
+    public InventoryPageSteps checkIfInventoryPageIsOpened() {
+        inventoryPage = new InventoryPage();
+        inventoryPage.getInventoryPageGrid().shouldBe(visible);
+        return this;
+    }
+
+    public void checkDescriptionOnUI(ItemsEnum itemName, String material) {
+        Boolean containsMaterial = containsMaterialInDescriptionOnUI(itemName, material);
+        Boolean isMadeFromCotton = itemName.getIsMadeFromCotton();
+        assertThat(containsMaterial).isEqualTo(isMadeFromCotton);
```

Comment:
not sure it's a good idea to verify two boolean values when you deal with Text. Because you are not able to see the Actual result. AssertJ returns text message - Expecting value to be true but was false, better to have Expected And Actual Texts